src/breadcrumbs/breadcrumbs.macros.html

```diff
@@ -1,17 +1,37 @@
-{% from '../icons/icons.macros.html' import icon %}
+{#
+  * @overview Macros for Breadcrumbs
+  * @module Breadcrumbs
+#}
+{% from 'nightshade-core/src/icons/icons.macros.html' import icon %}
+{% from 'nightshade-core/src/links/links.macros.html' import link %}
 
-{% macro breadcrumbs() %}
-<div class="breadcrumbs">
-  <ol class="breadcrumb-list">
-    <li class="breadcrumb-item ">
-      <a class="-light" href="/">
-        {{ icon("casper-logo-mark", size="medium" )}}
+{#
+  * Renders Breadcrumbs
+  * @param {array} data - List of links for Breadcrumbs
+  * @param {string} [color=light] - Indicates the color theme for the Breadcrumbs
+  * @param {string} [class] - Class name for style or JS targeting
+  * @param {string} [id] - ID for JS targeting
+#}
+
+{% macro breadcrumbs(data, color=null, class=null, id=null) %}
+<div class="breadcrumbs {{ color }}{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %}>
```

Comment:
@ckeisato The param doc says default is 'light', but no default is being set. Would be good to use BEM syntax: `breadcrumbs--light`, and this will help with organization of the sass too